Library/Formula/miller.rb

```diff
@@ -0,0 +1,30 @@
+class Miller < Formula
+  desc "Miller is like sed, awk, cut, and sort, and join for data such as CSV."
+  homepage "http://johnkerl.org/miller/"
+  url "https://github.com/johnkerl/miller/archive/v2.1.3.tar.gz"
+  sha256 "feb246c1a199e9a18c9ec630ddc021fca4ba9f1bf55286bf03018570b81e760c"
+
+  depends_on "asciidoc"
+
+  def install
+    ENV.deparallelize
+
+    system "make", "-C", "c", "mlr"
+    system "make", "manpage", "XML_CATALOG_FILES=/usr/local/etc/xml/catalog"
```

Comment:
Use `etc` instead of `/usr/local` here; see https://github.com/Homebrew/homebrew/blob/master/Library/Formula/reposurgeon.rb#L34 for an example.